Fibonacci.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Fibonacci {
+  public static void main(String[] args) {
+    int i = 0;
+    double FibNumber = 0;
+    int MaxNumb = 2147483647;
+    double sqrt5 = Math.sqrt(5);
+    double GoldenRatio = (1 + sqrt5) / 2;
+    Scanner input = new Scanner(System.in);
+    System.out.println("Enter a positive number to 2147483647: ");
+    int number = input.nextInt();
+    if (number >= 0 && number <= MaxNumb) {

```

Comment:
number <= MaxNumb is useless input.nextInt() never return something bigger than INT_MAX